src/libasr/pass/intrinsic_functions.h

```diff
@@ -2075,10 +2075,14 @@ namespace Dim {
 namespace Sqrt {
 
     static ASR::expr_t *eval_Sqrt(Allocator &al, const Location &loc,
-            ASR::ttype_t* arg_type, Vec<ASR::expr_t*> &args, diag::Diagnostics& /*diag*/) {
+            ASR::ttype_t* arg_type, Vec<ASR::expr_t*> &args, diag::Diagnostics& diag) {
         ASRUtils::ASRBuilder b(al, loc);
         if (is_real(*arg_type)) {
             double val = ASR::down_cast<ASR::RealConstant_t>(expr_value(args[0]))->m_r;
```

Comment:
Unrelated to your PR:
```suggestion
            double val = ASR::down_cast<ASR::RealConstant_t>(args[0])->m_r;
```
I think `expr_value()` is already computed and passed in `args`. So, we need not compute it here again. We should update this for other `eval_*()` functions as well (But in a separate PR).